tools/memap.py

```diff
@@ -10,10 +10,11 @@
 import argparse
 from prettytable import PrettyTable
 
-from tools.utils import argparse_filestring_type, \
```

Comment:
You cannot correctly drop the `tools.` from the front of this import. Please undo this change, as it has nothing to do with the feature being implemented.